build.gradle

```diff
@@ -52,18 +52,18 @@ subprojects {
     }
 
     dependencies {
-        compile "com.google.http-client:google-http-client-xml:1.20.0"
+        compile "com.google.http-client:google-http-client-xml:1.24.1"
         compile "com.google.guava:guava:25.1-jre"
         compile "com.squareup.okhttp3:okhttp:3.13.1"
-        compile "com.squareup.okio:okio:1.17.3"
+        compile "com.squareup.okio:okio:1.17.2"
```

Comment:
Any reason to downgrade `okio`?

Reply:
Well, okhttp is using okio 1.17.2 I thought that it would be better to downgrade than have newer version forced.

Reply:
ok. if possible we could move to okhttp 3.14.2

Reply:
I've checked it also with that version and tests were green but I didn't want to change too much in this commit. Moreover, 3.14.2 also uses 1.17.2 version of okio.